kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/internal/admin/AdminHttpInitializer.java

```diff
@@ -16,6 +17,7 @@
 
 public class AdminHttpInitializer extends ChannelInitializer<SocketChannel> {
 
+    private static final String HEALTH = "/healthz";
```

Comment:
I guess you're following the kube API server naming? It looks like this is long deprecated in favour of splitting into separate `livez` and `readyz` endpoints. So I guess we could call this `livez`?

Reply:
I'll do that.